ConfigurationHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace DatabaseConnectionProvider
+{
+    public class ConfigurationHandler
+    {
+        private const string MongoConnectionStringKey = "MongoConnection:ConnectionString";
+        private const string MongoDatabaseKey = "MongoConnection:Database";
+        
+        private readonly IConfiguration _configuration;
+        
+        public ConfigurationHandler(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public void ConfigureDataBaseSettings(IServiceCollection services)
+        {
+            services.Configure<MongoDbSettings>(options => {
+                options.ConnectionString = _configuration.GetSection(MongoConnectionStringKey).Value;
```

Comment:
Jeśli dobrze pamiętam do odwołanie się do value w przypadku nulla w konfiguracji sypnie, sprawdź to, niżej to samo

Reply:
No, I checked. I think that this works similar to gradle configuration. If there is no given key, it will be just an empty string.

Reply:
What will happen if _configuration.GetSection(someKey) == null and .Value will be requested?

Reply:
It will return empty String. I checked twice.